extensions/positron-python/package.json

```diff
@@ -647,7 +647,7 @@
                     "type": "string"
                 },
                 "python.interpreter.infoVisibility": {
-                    "default": "never",
+                    "default": "onPythonRelated",
```

Comment:
This will show the python interpreter path whenever a Python file is opened. There's also the option to always show it, but that might be confusing to sole-R users.

![Uploading Screenshot 2025-03-05 at 12.38.53 PM.png…]()

Reply:
Agreed on the default. Python users can always override the setting if they want